app/src/main/res/values/preloaded_fonts.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <array name="preloaded_fonts" translatable="false">
+        <item>@font/archivo_black</item>
+        <item>@font/blinker_bold</item>
+        <item>@font/candal</item>
         <item>@font/emblema_one</item>
+        <item>@font/russo_one</item>
```

Comment:
Double check you declared/imported these in the font folder. It's preventing me from building the app since they are missing.